packages/shylu/shylu_dd/frosch/src/CoarseSpaces/FROSch_CoarseSpace_decl.hpp

```diff
@@ -138,19 +138,19 @@ namespace FROSch {
         CommPtr MpiComm_;
         CommPtr SerialComm_;
         
-        ConstXMapPtrVec UnassembledBasesMaps_;
-        ConstXMapPtrVec UnassembledBasesMapsUnique_;
+        ConstXMapPtrVec UnassembledBasesMaps_ = ConstXMapPtrVec(0);
```

Comment:
That _should_ be what the default constructor of this class does, so you don't need the member initializer here.  (You don't explicitly initialize `MpiComm_` above, and you don't need to, so why initialize this one explicitly?)